azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/SupportedLanguages.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.configuration;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+final class SupportedLanguages {
```

Comment:
Why is this class in Java? Intended to be public API? Then it is missing `public` modifier.